Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -147,6 +147,11 @@ void HiresTexture::Update()
 
 void HiresTexture::Clear()
 {
+  if (s_prefetcher.joinable())
```

Comment:
Could you modify Shutdown to call Clear?

Reply:
Ha, didn't even notice that, will do!